service/create/templates.go

```diff
@@ -137,7 +137,7 @@ USERDATA_FILE={{.MachineType}}
     --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \
     --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \
     --trust-keys-from-https \
-    quay.io/coreos/awscli:025a357f05242fdad6a81e8a6b520098aa65a600 -- aws s3 --region {{.Region}} cp s3://{{.S3DirURI}}/$USERDATA_FILE /var/run/coreos/temp.txt
+    quay.io/coreos/awscli:025a357f05242fdad6a81e8a6b520098aa65a600 -- aws s3 --region {{.Region}} cp s3://{{.S3URI}}/$USERDATA_FILE /var/run/coreos/temp.txt
```

Comment:
+1 we need to start thinking about this. I think we're OK in this case. The S3 path is in the small cloudconfig on the master node and workers launch config.

When we support scaling up and down via the API we can use the existing value. Upgrading worker nodes will be trickier. If the operator creates a new bucket using the cluster ID we're OK. Otherwise we might need to do a manual migration :(